components/features.tsx

```diff
@@ -35,8 +35,8 @@ export default function Features() {
           {/* Section header */}
           <div className="max-w-3xl mx-auto text-center pb-12 md:pb-16">
             <h1 className="h2 font-semibold text-white mb-4">How it works</h1>
-            <p className="text-lg md:text-xl text-white">Instantly create dynamic and personalized study materials using your PDFs. 
-            <br/> Make studying a delight with our user-friendly app. </p>
+            <p className="text-lg md:text-xl text-white mb-0">Automatically create decks of flashcards by simply uploading your PDFs. 
+            <br/> Make studying enjoyable with our user-friendly app and features. </p>
```

Comment:
idk if enjoyable is the right word. is there anything that conveys reducing unnecessary pain and complexity? i can't think of a good word